### ISSUE TITLE
julia-0.5.1: Build fails

### ISSUE THREAD

**Issue by mnacamura** [NixOS/nixpkgs#27438]
## Issue description

```
LoadError("sysimg.jl",152,LoadError("show.jl",1238,ErrorException("error compiling Type: error compiling Type: error compiling compile: error compiling compile: could not load library \"libpcre2-8\"\ndlopen(libpcre2-8.dylib, 1): image not found")))
*** This error is usually fixed by running `make clean`. If the error persists, try `make cleanall`. ***
make[1]: *** [Makefile:229: /private/tmp/nix-build-julia-0.5.1.drv-0/julia-0.5.1/usr/lib/julia/sys.o] Error 1
make: *** [Makefile:99: julia-sysimg-release] Error 2
builder for ‘/nix/store/aw3grmaqz2jfh34z2i33xzkqbp6bz7jn-julia-0.5.1.drv’ failed with exit code 2
cannot build derivation ‘/nix/store/1jkpp8yh9j24f5g9nmd92aazgvbb28yq-julia-0.5.1-env.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/1jkpp8yh9j24f5g9nmd92aazgvbb28yq-julia-0.5.1-env.drv’ failed
```

### Steps to reproduce

```
nix-env -f '<nixpkgs>' -iA julia_05
```

## Technical details

* System: macOS Sierra
* Nix version: 1.11.12
* Nixpkgs version: "17.09pre-git"
* Sandboxing enabled: false

**Comment by vbgl** [comment]
It seems that hydra manages to build this package. It is available in the binary cache of the current (nixpkgs-17.09pre110532.50831d543d) state of the `nixpkgs-unstable` channel.

Maybe it’s related to sandboxing.

**Comment by mnacamura** [comment]
`nix-env -f '<nixpkgs>' -iA julia_05 --option build-use-sandbox true` on master fails with the same error message.

**Comment by vbgl** [comment]
Fixed by efad0d5e3fe4380b5276f41e38831387de9d371f.
